python/lsst/cp/pipe/ptc.py

```diff
@@ -618,6 +628,8 @@ def measureMeanVarCov(self, exposure1, exposure2, region=None, covAstierRealSpac
         #  Clipped mean of images; then average of mean.
         mu1 = afwMath.makeStatistics(im1Area, afwMath.MEANCLIP, im1StatsCtrl).getValue()
         mu2 = afwMath.makeStatistics(im2Area, afwMath.MEANCLIP, im2StatsCtrl).getValue()
+        if np.isnan(mu1) or np.isnan(mu2):
+            return np.nan, np.nan, np.nan
```

Comment:
This function claims to return `float, float, list`, so either change the doc or maybe make this return an empty list if that works for other places as a sentinel value.